hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -169,7 +184,7 @@ Default: 0
 Version: pieeprom-2020-04-16.bin  
 
 ### SELF_UPDATE
-Allows the bootloader to update itself instead of requiring recovery.bin. This is intended to make it easier to update the bootloader firmware via network boot. To enable set SELF_UPDATE=1 and add bootloader_update=1 in config.txt.
+Allows the bootloader to update itself instead of requiring recovery.bin. This is intended to make it easier to update the bootloader firmware via network boot. To enable set SELF_UPDATE=1 and add **bootloader_update=1** in config.txt.
 N.B. There is no automatic rollback in the event of a power failure during the firmware update. In the unlikely event of this happening you will have to use Pi Imager to apply the rescue image.
```

Comment:
We generally use `backticks` rather than **highlighting** for this sort of thing.

Reply:
done